### DISCUSSION TITLE
화장실 리뷰 앱

### DISCUSSION THREAD

**Discussion by ypd01018** [ondal-house/zu-story#14] [아이디어 선정 안건]
- 호텔, 식당 등에 위치한 화장실 청결상태 리뷰앱
- 공중화장실 위치 공유

Pros
- 간단하게 화장실에 대한 정보 쓰기, 읽기 등 CRUD 형태만 구현하면 가능할 것 
- 마찬가지로 위치 기반 서비스 구현을 해볼 수 있을 것

Cons
- 너무 간단한 프로젝트가 되어 얻어가는 것이 많이 없을수도..

**Comment by gychoi0916** [comment]
1) q. 공중 / 개방 화장실 -> 네이버 지도에도 뜸 . 개인 건물의 경우 위치를 얻는 것이 어려울 것 .(ㅋ 네이버도 실패한)
개인이 등록한다면 법적 문제는 없는지 , 
2)
